package.json

```diff
@@ -61,16 +61,19 @@
     ]
   },
   "dependencies": {
+    "@glimmer/syntax": "^0.83.1",
     "@lint-todo/utils": "^12.0.1",
     "chalk": "^4.1.2",
     "ci-info": "^3.3.0",
     "date-fns": "^2.27.0",
+    "ember-template-imports": "^2.0.0",
     "ember-template-recast": "^6.1.0",
     "find-up": "^6.2.0",
     "fuse.js": "^6.5.3",
     "get-stdin": "^9.0.0",
     "globby": "^12.0.2",
     "is-glob": "^4.0.3",
+    "magic-string": "^0.25.7",
```

Comment:
this'll be removed once: https://github.com/ember-template-imports/ember-template-imports/pull/30
is merged and released